src/util/api/index.js

```diff
@@ -269,3 +269,6 @@ export const formatFeaturedNFTUrl = wallet =>
 
 export const formatHiddenNFTUrl = wallet =>
   `/api/v2/users/${wallet}/nfts/hidden`;
+
+export const formatUnhiddenNFTUrl = wallet =>
+  `/api/v2/users/${wallet}/nfts/normal`;
```

Comment:
I suggest combining the above three functions
```suggestion
export const getNFTDisplayStateURL = (wallet, state) =>
  `/api/v2/users/${wallet}/nfts/${state}`;
```